src/model/Model.lua

```diff
@@ -210,6 +209,70 @@ function Model:printModel()
     end
 end
 
+local function getSubjectHierarchyMap(policies)
+    local subjectHierarchyMap = { }
+    --Tree structure of role
+    local policyMap = { }
+    for _, policy in pairs(policies) do
+        if #policy < 3 then
```

Comment:
I think this should be `#policy < 2`, since 2 values in a grouping policy is a valid policy. This would then work for policies without domains too.

Reply:
Got it，fixed.